controller/rest/repository.go

```diff
@@ -22,7 +22,7 @@ import (
 
 const (
 	repositoryDefaultTag      = "latest"
-	maxRepoScanTasks          = 32
+	maxRepoScanTasks          = 4
```

Comment:
maxRepoScanTasks determines the maximum number of scan tasks that can run concurrently.
We've set it to 4 after observing that each scan task consumes approximately 500 MB of memory.
Running 32 concurrent scans could consume up to 16 GB, leading to Out-Of-Memory (OOM) issues.
Therefore, limiting it to 4 is a safer choice to prevent excessive memory usage.